src/js/components/Books/Books.js

```diff
@@ -2,22 +2,22 @@ import React, { Component } from 'react';
 import Navbar from '../Navbar/Navbar';
 import Bookcard from '../Books/Bookcard/Bookcard';
 
-import './Book.sass';
+import './Book.scss';
 class  Books extends Component {
   render() {
-    var loc_navBarTitle  = "BOOKS";
-    var loc_navbarItems = [true,true,true,true];
-
+    let loc_navBarTitle  = "BOOKS";
+      let loc_navbarItems = [true,true,true,true];
+      let rows = [];
+      for (let i = 0; i < 5; i++) {
```

Comment:
What is 5?
Length of anything?

Reply:
for display purposes, how many times I want a fragment to be shown as I want the html fragment to be loaded 5 times.
It is suppose to be read from the database but that is currently not up and running.

Reply:
You can put it in a const variable